### PR TITLE
Get voter limit from shared config

### PR DESCRIPTION
**Pull request by ArendPeter**
## Description
Update create-react-app to let us use shared files

## Screenshots / Videos (frontend only) 

image

image

image

### CODE REVIEW

**Comment by mikefranze** [comment]
Looks good, but I'd like to see that it works in the back end too in our deployment process before approving.

**Comment by mikefranze** [comment]
Have you tried building the docker image? I think you'll need to copy the shared folder like the domain folder.

**Comment by ArendPeter** [comment]
> Have you tried building the docker image? I think you'll need to copy the shared folder like the domain folder.

Done! I added the copied commands and tested building the image. It works

**Comment by ArendPeter** [comment]
Rebased, it should be a clean merge now

File: shared/SharedConfig.ts [hunk 1]
```diff
@@ -0,0 +1,9 @@
+export const sharedConfig = {
+    FREE_TIER_PRIVATE_VOTER_LIMIT: 2, // making really low at first so it's easy to test
```
Comment (mikefranze) [inline shared/SharedConfig.ts:2]: Can we increase this to 20? Would like to be able to test with more voters in private elections. We can lower it locally for testing.

Comment (mikefranze) [reply shared/SharedConfig.ts:2]: This is still only applied in the front end right? Where does the backend use this to reject the request?

Comment (ArendPeter) [reply shared/SharedConfig.ts]: You're right I messed up the rebase and lost the most recent commits. Should be fixed now

(more specifically I did the rebase on my desktop, but the latest version of the branch was from my laptop, so I overwrite the laptop changes with the force push)

**Review by mikefranze — APPROVED** [review]

**Pull request merged**